src/store/listAnime.js

```diff
@@ -53,6 +53,12 @@ export default [
         animeYear:1998,
         animeLink:"https://myanimelist.net/anime/185/Initial_D_First_Stage"
     },
+    {
+        animeEmoji:"👩🐉👩‍💼",
+        animeName:"Kobayashi-san Chi no Maid Dragon",
+        animeYear:2017,
+        animeLink:"https://myanimelist.net/anime/33206/Kobayashi-san_Chi_no_Maid_Dragon?q=Dragon%20Maid"
```

Comment:
i think we can remove the query string , 
that would be nice

Reply:
sure thing

Reply:
please send pr again @lordjbs

Reply:
So uh, new or?

Reply:
pr*

Reply:
just commit again to this pr

Reply:
Done